blocks/press-cards/press-cards.js

```diff
@@ -76,6 +76,7 @@ export default async function decorate(block) {
     });
     ul.append(li);
   });
+  ul.querySelectorAll('img').forEach((img) => img.closest('picture').replaceWith(createOptimizedPicture(img.src, img.alt, false, [{ width: '750' }])));
```

Comment:
What's the issue that we are fixing here?

Reply:
The href is not working for the press cards images

Reply:
Thanks. But href still does not seem to work for me for [this page](https://issue-fix-image-download--zeiss--hlxsites.hlx.page/de/semiconductor-manufacturing-technology/news-und-events/smt-pressemeldung/ausgezeichnete-ausbildung-bei-zeiss).

Reply:
> @sdmcraft now it should work

Thanks. One minor change needed though. On the original site, the download happens in a new tab while in our case, it happens in the same tab.